go/libkb/chain_link_v2.go

```diff
@@ -141,12 +141,11 @@ func (t SigchainV2Type) RequiresAtLeastRole() keybase1.TeamRole {
 	if !t.IsSupportedTeamType() {
 		// Links from the future require a bare minimum.
 		// They should be checked later by a code update that busts the cache.
-		return keybase1.TeamRole_READER
+		return keybase1.TeamRole_BOT
```

Comment:
bots sign links?

Reply:
bots cant sign but this is used to verify links in `TeamLoader.verifyLink`

Reply:
`verifyLink` only handles `BOT` because it's output here `RequiresAtLeastRole`

Reply:
actually bots have to be able to sign leave links, can make the default for unsupported back to reader though